src/utils/env.js

```diff
@@ -38,4 +38,6 @@ module.exports = (NODE_ENV = process.env.NODE_ENV) => {
             // });
         }
     });
+
+    return process.env;
```

Comment:
just double checking : it's fine in RatesFeeder and Matchmaker, the just don't use the loadenv fx return value yet?

Reply:
yepp :)